src/com/komiamiko/fcorbit/Main.java

```diff
@@ -34,6 +34,11 @@
  */
 public class Main {
 	
+	/**
+	 * Minimum number of pixels movement to register a mouse drag
+	 */
+	public static final int MOUSE_DRAG_MIN_PIXELS = 4;
```

Comment:
Remember to check back on this if sensitivity issues should ever arrive.

Reply:
I know.